config.json

```diff
@@ -0,0 +1,105 @@
+{
+  "files": [
```

Comment:
that's exactly why I chose json, easy enough to parse and easy to replace. This is actually something I meant to mention in my pr. What's standard around here?